apache/_docroot.shtml

```diff
@@ -5,15 +5,26 @@
         <tt>docroot</tt> in your repository to serve content from. This is a common pattern, but many projects have their own
         preference. If your document root is in a different location you have a couple of different options.
     </p>
-    <h3>1. Move your content to /docroot in your repository.</h3>
-    <p>This is the most straightforward fix for Tugboat, but can be problematic if your repository has already been deployed
-        elsewhere. If you choose this option, be sure to follow up on other deployments to make sure things still work.
+    <h3>Option 1. Add a <tt>docroot/</tt> folder to your repository</h3>
+    <p>
+      This is the most straightforward fix for Tugboat, but can be problematic if your repository has already been 
+      deployed elsewhere. If you choose this option, create a <tt>docroot</tt> folder and move your existing content 
+      you want Apache to serve into that folder. Be sure to follow up on other deployments to make sure things still 
```

Comment:
This sentence does not flow well: "move your existing content you want Apache to serve..."

I also think I'd like to avoid mentioning Apache, and keep this page generic so it highlights Tugboat instead of any underlying technologies.